front/src/Enum/EnvironmentVariable.ts

```diff
@@ -1,30 +1,46 @@
-const DEBUG_MODE: boolean = process.env.DEBUG_MODE == "true";
-const START_ROOM_URL: string =
-    process.env.START_ROOM_URL || "/_/global/maps.workadventure.localhost/Floor1/floor1.json";
-const PUSHER_URL = process.env.PUSHER_URL || "//pusher.workadventure.localhost";
-export const ADMIN_URL = process.env.ADMIN_URL || "//workadventu.re";
-const UPLOADER_URL = process.env.UPLOADER_URL || "//uploader.workadventure.localhost";
-const ICON_URL = process.env.ICON_URL || "//icon.workadventure.localhost";
-const STUN_SERVER: string = process.env.STUN_SERVER || "stun:stun.l.google.com:19302";
-const TURN_SERVER: string = process.env.TURN_SERVER || "";
-const SKIP_RENDER_OPTIMIZATIONS: boolean = process.env.SKIP_RENDER_OPTIMIZATIONS == "true";
-const DISABLE_NOTIFICATIONS: boolean = process.env.DISABLE_NOTIFICATIONS == "true";
-const TURN_USER: string = process.env.TURN_USER || "";
-const TURN_PASSWORD: string = process.env.TURN_PASSWORD || "";
-const JITSI_URL: string | undefined = process.env.JITSI_URL === "" ? undefined : process.env.JITSI_URL;
-const JITSI_PRIVATE_MODE: boolean = process.env.JITSI_PRIVATE_MODE == "true";
+declare global {
+    interface Window {
+        env?: Record<string, string>;
+    }
+}
+
+const getEnv = (key: string): string | undefined => {
+    if (!window.env) {
```

Comment:
```suggestion
    if (!window || !window.env) {
```

Maybe a simple fix for Jasmine.

Reply:
Checking for `!global.window` seems to work. Nice idea!

One thing to note though:
Jasmine will now always use the default values. I might just add a case to return the value from `process.env` if it is set.